lib/rpmds.h

```diff
@@ -82,12 +82,16 @@ typedef rpmFlags rpmsenseFlags;
     _notpre(RPMSENSE_SCRIPT_PREUN|RPMSENSE_SCRIPT_POSTUN)
 #define _UNORDERED_ONLY_MASK \
     _notpre(RPMSENSE_RPMLIB|RPMSENSE_CONFIG|RPMSENSE_PRETRANS|RPMSENSE_POSTTRANS|RPMSENSE_SCRIPT_VERIFY)
+#define _STILL_ORDERED_MASK \
+    _notpre(RPMSENSE_SCRIPT_PRE|RPMSENSE_SCRIPT_POST|RPMSENSE_SCRIPT_PREUN|RPMSENSE_SCRIPT_POSTUN)
```

Comment:
Calling this _ORDERED_ONLY_MASK would seem more in line with the others.

Reply:
Although okay, calling it that might also be misleading since it'll miss the case where no flags at all have been set.

Reply:
...but then I'm not sure calling it _STILL_ORDERED_MASK makes that aspect any clearer.